data/tosca/yorc-slurm-types.yml

```diff
@@ -97,22 +97,32 @@ node_types:
           Set a limit on the total run time of the job allocation.
           Time formats include "minutes", "minutes:seconds", "hours:minutes:seconds", "days-hours", "days-hours:minutes" and "days-hours:minutes:seconds"
         required: false
-      batch:
-        type: boolean
-        description: True if the job is run on batch mode.
-        required: false
-        default: true
-      extra_options:
+      extra_job_options:
```

Comment:
Maybe it's time to group properties functionally:

- using data types if possible
- or at least in giving them in a coherent order (in a4c plugin)